src/store/actions/performNextAction/swap.js

```diff
@@ -72,11 +72,33 @@ async function initiateSwap ({ getters, dispatch }, { order, network, walletId }
   }
 }
 
+async function fundSwap ({ getters, dispatch }, { order, network, walletId }) {
+  if (await dispatch('checkIfQuoteExpired', { network, walletId, order })) return
+
+  const toClient = getters.client(network, walletId, order.to)
+
+  const fundTx = await toClient.swap.fundSwap(
+    order.fromFundHash,
+    order.fromAmount,
+    order.fromCounterPartyAddress,
+    order.fromAddress,
+    order.secretHash,
+    order.swapExpiration,
+    order.fee
+  )
+
+  console.log('ORDER FUNDED', fundTx)
+
+  return {
+    status: 'INITIATION_REPORTED'
```

Comment:
The tx obj also needs to be committed to state so that later it can be used to bump fee

Reply:
got it

Reply:
fixed